data/test/scenarios/swarm_disables_upgrades.cfg

```diff
@@ -82,17 +96,20 @@
             variable=a
             kill=yes
         [/store_unit]
+
         [store_unit]
             [filter]
                 id=bob
             [/filter]
             variable=b
         [/store_unit]
+
         [unstore_unit]
             variable=a
             find_vacant=yes
             x,y=$b.x,$b.y
         [/unstore_unit]
+
```

Comment:
I've been wondering about changing some tests from `GENERIC_UNIT_TEST` to `COMMON_KEEP_A_B_UNIT_TEST`, thus removing the need to move Alice around. Haven't started work on that, but if you're editing spacing it might be better to replace the code too.

Reply:
I'm not sure how they're related? I was just adding some spacing to make it easier for me to read.

Reply:
Just that that refactor would remove these lines (along with a lot of the code).